### PR TITLE
Fix issue 422

### PR DESCRIPTION
**Pull request by mhrobinson**
Fix tab not working for all the commands

Signed-off-by: Mike Robinson <mikerob@cisco.com>

### CODE REVIEW

**Comment by mhrobinson** [comment]
Currently extending the fix to include auto completion for all the command options as well.

File: scripts/contrib/completion/bash/netctl [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
     network\
     tenant\
     policy\
+    external-contracts\
     global\
+    aci-gw\
+    bgp\
+    app-profile\
+    service\
     help"
```
Comment (rchirakk) [inline scripts/contrib/completion/bash/netctl]: Can you keep this list sorted ?

Comment (dseevr) [inline scripts/contrib/completion/bash/netctl]: This is actually the current order of the command output from `netctl --help`

We need to reorder the `Commands` array in `netctl/commands.go`, too.

Comment (mhrobinson) [inline scripts/contrib/completion/bash/netctl]: Should we re-order bash/netctl and netctl/commands.go in a separate fix after this to keep this diff readable?

Comment (dseevr) [inline scripts/contrib/completion/bash/netctl]: That sounds good.  You weren't included in that "we" FYI.  That's our problem  😄

**Comment by mhrobinson** [comment]
Updated the diff to include completion for all the command options as well.

File: scripts/contrib/completion/bash/netctl (hunk 1 above)
Comment (rchirakk) [inline scripts/contrib/completion/bash/netctl]: ```>>re-order netctl/commands.go ```
done, https://github.com/contiv/netplugin/pull/848

Comment (mhrobinson) [inline scripts/contrib/completion/bash/netctl]: The completion script automatically sorts the completions when it displays them, the same as your above fix for the help.  So unless we re-order commands.go, it seems to make sense to leave this script in the original matching order.  All the bash functions in the script are also laid out the original order, so ideally we would then need to alphabetize the entire file.

```text
[vagrant@netplugin-node1 netplugin]$ netctl 
aci-gw              external-contracts  netprofile          tenant
app-profile         global              network             version
bgp                 group               policy              
endpoint            help                service             
```

Comment (rchirakk) [inline scripts/contrib/completion/bash/netctl]: A sorted list would be easier to compare with netctl --help to check missing  commands

Comment (mhrobinson) [inline scripts/contrib/completion/bash/netctl]: That is a valid point. :)  I've re-ordered the command lists in the completion script to match the help output.

**Comment by mhrobinson** [comment]
Updated the diff to re-order the completion script command lists to match the help output.

**Pull request merged**